.github/workflows/deploy.yml

```diff
@@ -30,6 +30,21 @@ jobs:
         rm -f "$TEMP_DEB"
         apm --version
 
+    - name: Prepare grammar file & push updated release # Has to be before Publish step
+      run: |
+        git config --global user.name "github-actions[bot]"
+        git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
I don't think it matters what this is set to (I used `action@github.com` on [cylc/Cylc.tmbundle](https://github.com/cylc/Cylc.tmbundle/blob/master/.github/workflows/build.yml#L39-L40)), interested where `41898282` came from though?

Reply:
These are generated by github, yer? I have a no-reply email set up on my GH account, so MO email not scraped by bots for spam.

Reply:
It's from https://github.community/t/push-built-files-to-repo-using-actions/18186/4

When the commits show up on here, clicking the author takes you to the landing site for GitHub actions. Not needed, but it's a nice touch

Reply:
https://github.com/cylc/language-cylc/commit/d4609e93d816394b8b5b31a7712a41e795269994

Reply:
Ah, so it's mimicking a GH no-reply. Because we've used 41898282 these commits will be associated with any other made by other people using the same number?

Reply:
I tried searching `author:github-actions[bot]` in all of github but our one didn't show up for whatever reason.